src/SLCore.UnitTests/Core/SLCoreListenerSetUpTests.cs

```diff
@@ -21,7 +21,7 @@
 using SonarLint.VisualStudio.SLCore.Core;
 using SonarLint.VisualStudio.TestInfrastructure;
 
-namespace SonarLint.VisualStudio.SLCore.UnitTests
+namespace SonarLint.VisualStudio.SLCore.UnitTests.Core
```

Comment:
moved tests to core folder so it would have same structure with the main project. It's 2 tasks in 1 PR but since PR is so small I think it's OK.